client/app/dbaas/logs/detail/streams/follow/streams-follow.html

```diff
@@ -1,17 +1,51 @@
-<div class="cui-page__content log-streams-follow-live">
+<div class="cui-page__content log-streams-follow">
     <oui-back-button data-state="dbaas.logs.detail.streams"
         class="back_button_inline_text"
         data-state-params="{ serviceName: ctrl.serviceName}">
     </oui-back-button>
     <span class="oui-header_3" data-translate="logs_streams_follow_live"></span>
-    <h6 class="oui-header_6">{{ctrl.stream.info.title}}</h6>
+    <h6 class="oui-header_6 pb-3">{{ctrl.stream.data.info.title}}</h6>
 
-    <oui-action-menu text="{{ ::'common_actions' | translate}}" align="start">
-        <oui-action-menu-item text="{{ ::'logs_streams_follow_copy_address' | translate}}" on-click="ctrl.copyAddress()"></oui-action-menu-item>
-        <oui-action-menu-item text="{{ ::'logs_streams_follow_download_client' | translate}}" href="https://github.com/ovh/ldp-tail/releases" external></oui-action-menu-item>
-        <oui-action-menu-item text="{{ ::'logs_streams_follow_close_connection' | translate}}" on-click="ctrl.closeConnection()"></oui-action-menu-item>
-        <oui-action-menu-item text="{{ ::'logs_streams_follow_open_connection' | translate}}" on-click="ctrl.openConnection()"></oui-action-menu-item>
-    </oui-action-menu>
+    <oui-spinner data-ng-show="ctrl.stream.loading"></oui-spinner>
+    <div data-ng-hide="ctrl.stream.loading">
 
+        <div>
+            <p data-ng-hide="ctrl.isConnectionClosed()"><strong>{{'logs_streams_connection_opened' | translate}} </strong>
+                <span data-ng-show="ctrl.getLastEvent()"> - {{ctrl.getLastEvent() * 1000 | momentFormat:'LL' }} {{ctrl.getLastEvent() * 1000 | momentFormat:'LTS' }}</span>
+            </p>
+            <p data-ng-hide="ctrl.isConnectionClosed()"><strong>{{'logs_streams_displaying_logs' | translate}}</strong>
+                <span> - {{ctrl.getTotalMessages() | number}}/1000</span>
```

Comment:
For maintainability purpose, you should put this number (1000) in a constant so we know what it is for.

Reply:
good catch. I missed to put it in constants.